pom.xml

```diff
@@ -1,26 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project> xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.example</groupId>
     <artifactId>LoopFromHackerRank</artifactId>
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>16</maven.compiler.source>
-        <maven.compiler.target>16</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <springframework.boot.version>2.5.5</springframework.boot.version>
     </properties>
```

Comment:
Java version was set to `16`. Most companies are using versions 8 through 11. There's not a huge difference b/w them. 9 or 10 introduces streams. and I think there's some loosely typed variable stuff..

also, you'll see below that I added a `dependency` management block to pull in `spring-boot` dependencies. It was originally referencing a `parent` pom - which is part of the demo, you don't have that, so I'm referencing actual spring

Reply:
Do I need to change my jdk version to run this now?